src/app/ui/components/items/item.stories.tsx

```diff
@@ -0,0 +1,147 @@
+  component: Item.Root,
+  tags: ['autodocs'],
+  title: 'Item',
+};
+
+export default meta;
+type Story = StoryObj<typeof Item.Root>;
+
+export const Default: Story = {
+  render: () => (
+    // eslint-disable-next-line no-console
+    <Item.Root>
+      <Item.Content>
+        <ItemDefaultLayout
+          flagImg={<BtcIcon size="40px" />}
+          contentLeftTop={<Item.Text>Label</Item.Text>}
+          contentLeftBottom={<Item.Caption textStyle="caption.01">Caption</Item.Caption>}
+          contentRightTop={<Item.Text>1,000.00 ABC</Item.Text>}
+          contentRightBottom={<Item.Caption textStyle="caption.02">$1,000.00</Item.Caption>}
+        />
+      </Item.Content>
+    </Item.Root>
+  ),
+};
+
+export const DefaultPressable: Story = {
+  render: () => (
+    // eslint-disable-next-line no-console
+    <Item.Root isPressable onClick={() => console.log(`I'm interactive`)}>
+      <Item.Content>
+        <ItemDefaultLayout
+          isPressable
+          flagImg={<BtcIcon size="40px" />}
+          contentLeftTop={<Item.Text>Label</Item.Text>}
+          contentLeftBottom={<Item.Caption textStyle="caption.01">Caption</Item.Caption>}
+          contentRightTop={<Item.Text>1,000.00 ABC</Item.Text>}
+          contentRightBottom={<Item.Caption textStyle="caption.02">$1,000.00</Item.Caption>}
+        />
+      </Item.Content>
+    </Item.Root>
+  ),
+};
+
+export const Selected: Story = {
+  render: () => (
+    // eslint-disable-next-line no-console
+    <Item.Root isPressable isSelected onClick={() => console.log(`I'm interactive`)}>
+      <Item.Content>
+        <ItemDefaultLayout
+          isPressable
+          flagImg={<BtcIcon size="40px" />}
+          contentLeftTop={
+            <HStack gap="space.01">
+              <Item.Text>Label</Item.Text>
+              <CheckmarkIcon />
+            </HStack>
+          }
+          contentLeftBottom={<Item.Caption textStyle="caption.01">Caption</Item.Caption>}
+          contentRightTop={<Item.Text>1,000.00 ABC</Item.Text>}
+          contentRightBottom={<Item.Caption textStyle="caption.02">$1,000.00</Item.Caption>}
+        />
+      </Item.Content>
+    </Item.Root>
+  ),
+};
+
+export const Disabled: Story = {
+  render: () => (
+    // eslint-disable-next-line no-console
+    <Item.Root isDisabled isPressable onClick={() => console.log(`I'm interactive`)}>
+      <Item.Content>
+        <ItemDefaultLayout
+          isDisabled
+          isPressable
+          flagImg={<BtcIcon size="40px" />}
+          contentLeftTop={<Item.Text>Label</Item.Text>}
+          contentLeftBottom={<Item.Caption textStyle="caption.01">Caption</Item.Caption>}
+          contentRightTop={<Item.Text>1,000.00 ABC</Item.Text>}
+          contentRightBottom={<Item.Caption textStyle="caption.02">$1,000.00</Item.Caption>}
+        />
+      </Item.Content>
+    </Item.Root>
+  ),
+};
+
+export const WithButtons: Story = {
+  render: () => (
+    // eslint-disable-next-line no-console
```

Comment:
Quite the _nit_ but sometimes we don't need the `// eslint-disable-next-line no-console` , not a big deal though in a story